### ISSUE TITLE
Sort autocomplete items by similarity to query

### ISSUE THREAD

**Issue by eduxstad** [eduxstad/boiler-grades#3]
Currently the autocomplete component simply lists all items that match your query via the `filter` function. This could be improved to sort items by similarity (e.g. Fuzzy Searching/Levenshtein distance). See #2 for more details.

**Comment by kdkasad** [comment]
I don't actually think this is possible using `VAutoComplete`. That component simply takes a filter function, which tells it whether or not an individual item should appear in the autocomplete list. There isn't a way to return a ranking of items, which would be needed to sort items by similarity.

**Comment by eduxstad** [comment]
Yeah that unfortunately appears to be the case. [Link](https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/composables/filter.ts#L55). You'd probably have to modify/customize the component in some other way to accomplish this.
